ivy/functional/frontends/numpy/linear_algebra/matrix_and_vector_products.py

```diff
@@ -4,3 +4,13 @@
 
 def outer(a, b, out=None):
     return ivy.outer(a, b, out=out)
+
+
+#inner
```

Comment:
no need to import ivy again down here. Also remove the comments like #global and # inner. Just have 2 lines between outer and inner functions.